src/utils.js

```diff
@@ -385,3 +377,7 @@ export function deepProxy(target, proxyHandler) {
 function isNumeric(subject){
     return ! isNaN(subject)
 }
+
+export default {
+    arrayUnique,
+}
```

Comment:
We are already exporting that function and I'm not sure that arrayUnique should be the default for this file. It's probably tidier if we don't add these lines.
You can use `import { utils } from '../src/utils'` in your test

Reply:
I actually wanted to export a object with all named exports, as we can see on many projects. However my changes affect only arrayUnique and so I didn't export all... I still find more interesting export like I did because the test is an example where I (should) use all methods and in this case the import will be tidier, but I can change the import in test as well if you prefer

Reply:
It's more down to what Caleb prefers, but I think we just need to keep it consistent (one way or the other one).
This is the kind of thing that falls in the "opinion" category, another dev could say that they really love the other syntax but, at the end, there's really little difference between them. 
I personally don't have strong feeling pro or against both ways.

Reply:
@calebporzio can you help here? I agree with @SimoTod about it, so what you think? We'll avoid export default on the project?

Reply:
yeah, let's avoid it. We can stick with the current pattern we've established in utils.js. Thanks!